#!/usr/bin/python3
def raise_exception():
    divide = 5 / "a"
